Description: Unban Routes 2.0
 The Debian package works perfectly, so we accept it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-08-01

--- glance-11.0.0~b2.orig/requirements.txt
+++ glance-11.0.0~b2/requirements.txt
@@ -14,8 +14,7 @@ SQLAlchemy<1.1.0,>=0.9.7
 anyjson>=0.3.3
 eventlet>=0.17.4
 PasteDeploy>=1.5.0
-Routes!=2.0,!=2.1,>=1.12.3;python_version=='2.7'
-Routes!=2.0,>=1.12.3;python_version!='2.7'
+Routes
 WebOb>=1.2.3
 sqlalchemy-migrate>=0.9.6
 httplib2>=0.7.5
